tests/pr_test.go

```diff
@@ -54,6 +57,40 @@ func TestRunBasicExample(t *testing.T) {
 	assert.NotNil(t, output, "Expected some output")
 }
 
+func TestRunAdvanceExample(t *testing.T) {
+	t.Parallel()
+
+	options := testschematic.TestSchematicOptionsDefault(&testschematic.TestSchematicOptions{
+		Testing: t,
+		Prefix:  "advanced-key-protect",
+		TarIncludePatterns: []string{
+			"*.tf",
+			"modules/*/*.tf",
+		},
+		ResourceGroup:          resourceGroup,
+		TemplateFolder:         advancedExampleTerraformDir,
+		Tags:                   []string{"test-schematic"},
+		DeleteWorkspaceOnFail:  false,
+		WaitJobCompleteMinutes: 60,
+	})
+
+	// If "jp-osa" was the best region selected, default to us-south instead.
+	// "jp-osa" is currently not allowing hs-crypto be used for encrypting in that region.
+	if options.Region == "jp-osa" {
+		options.Region = "us-south"
+	}
```

Comment:
There is no need for this logic here as we are not using HPCS in this example

Reply:
@ocofaigh removed this logic

Reply:
@ocofaigh do have any clue on golang_lint, how can i fix it. It is also failing on my side even after updating the golang_lint